src/components/Table/utils/utils.ts

```diff
@@ -0,0 +1,72 @@
+import {TableColumnConfig} from '../Table';
+
+export interface DataItem {
+    name: string;
+    city?: string;
+    phone: string;
+    count: number;
+    date: string;
+    disabled?: boolean;
+}
+
+export const data: DataItem[] = [
```

Comment:
Why you moved the file? This should be placed inside `__stories__`, otherwise it will be compiled and packaged.